js/app.js

```diff
@@ -1,16 +1,28 @@
 // Enemies our player must avoid
-var Enemy = function() {
-    // Variables applied to each of our instances go here,
-    // we've provided one for you to get started
+// in this file we have to create a render methods for player and enemies
 
+var Enemy = class {
```

Comment:
- [ ] Please read the assignment again. JS Object oriented features were supposed to be used. Lets not use ES2015 for this assignment.